weather-wise/pom.xml

```diff
@@ -52,8 +53,9 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.10.1</version>
                 <configuration>
-                    <source>19</source>
-                    <target>19</target>
+                    <release>11</release>
+                    <source>11</source>
+                    <target>11</target>
                 </configuration>
```

Comment:
Downgraded maven compiler plugin version to accommodate systems without the latest Java version.